src/clr-angular/forms/combobox/combobox.ts

```diff
@@ -83,12 +83,13 @@ export class ClrCombobox<T> implements AfterContentInit, OnDestroy {
 
   private registerPopoverIgnoredInput() {
     if (this.input) {
-      this.ifOpenService.registerIgnoredElement(this.input);
+      // todo: intentionally commented; resolve while merging the Combobox
+      //this.toggleService.registerIgnoredElement(this.input);
```

Comment:
Same as above. I already have changes ready in my combobox branch for this, which I stashed so I can separate the popover only changes from the combobox changes. 
If I fix them now, it will be fix of the tests of some dead code. And I'll have to merge 2 changes over base instead of 1 change over base.

Reply:
When we have concerns like this, I think it helps to make a common comment anywhere you are commenting something out for a single purpose, then you can search for all of them easily. So update the comments where you've commented out something temporarily.

```
// @TODO COMBOBOX some additional comment
```